lib/ansible/module_utils/keycloak.py

```diff
@@ -53,7 +53,7 @@ def keycloak_argument_spec():
     """
     return dict(
         auth_keycloak_url=dict(type='str', aliases=['url'], required=True),
-        auth_client_id=dict(type='str', default='admin-cli'),
+        auth_client_id=dict(type='str'),
```

Comment:
This is a backwards incompatible change for the module, as the default is being removed. Is there a reason why this default should not be added to the docs instead?

Reply:
I initially wanted to document it but thought about why it would be the default. The only reason it is the default is that this client comes shipped with Keycloak as a do-all client (i.e. akin to root), and I had used it as an example in the docs elsewhere as something that would work.
This default was thus far undocumented, so I thought it would be a good opportunity to remove it entirely. However, I can document it instead as well. There should be no harm.

Reply:
Go ahead and update the docs then, so backwards compatibility is maintained.